src/main/webapp/organization.js

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+class Organization {
+  constructor(organization, isMaintainer = false) {
+    this.organization = organization;
+    this.isMaintainer = isMaintainer;
+    this.organizationElement = document.createElement("div");
+    this.createOrganization();
+  }
+
+  getOrganization() {
+    return this.organizationElement;
+  }
+
+  createOrganization() {
+    this.organizationElement.classList.add("organization");
+
+    const organizationNameElement = document.createElement('div');
+    organizationNameElement.classList.add("organization-name");
+    organizationNameElement.textContent = this.organization.name;
+
+    this.organizationElement.addEventListener('click', () => {
+      // TODO: GetOrganizationServlet to display more information about this servlet
+
+      const popupIndexPage = document.getElementById("organization-popup-area");
+      popupIndexPage.textContent = "";
+      popupIndexPage.appendChild(this.createOrganizationPopup());
+      popupIndexPage.classList.add("show-popup")
```

Comment:
need to also remove the classList hide-pop up here. This should be the same as the pull request that was approved but not merged